lib/vanagon/platform/dsl.rb

```diff
@@ -128,6 +128,13 @@ def cross_compiled(cxx_flag)
         @platform.cross_compiled = cxx_flag
       end
 
+      # Set the vendor_build_tools flag for the platform
+      #
+      # @param flag [Boolean] True if this platform uses host OS provided build tools
```

Comment:
The documentation for this method seems off/incomplete -- am I reading it poorly or is it misleading?